src/utils/cookie.ts

```diff
@@ -0,0 +1,23 @@
+import Cookies from 'universal-cookie';
+
+const cookie = new Cookies();
+
+export const getUnitPreferences = (): UnitPref => {
+    // const cookie = new Cookies();
+    let unitPref: UnitPref = { Temp: 'C', Depth: 'ft' };
```

Comment:
Let's use the same system for both units, i.e. C and m.

Reply:
Okay